main/webapp/modules/core/langs/translation-en.json

```diff
@@ -371,6 +371,9 @@
     "core-dialogs/name": "Name",
     "core-dialogs/action": "Action",
     "core-dialogs/should-return-number": "your expression should return a number",
+    "core-dialogs/remove-duplicate-rows": "Remove duplicate rows",
+    "core-dialogs/remove-duplicate-rows-col-select": "Select columns to identify duplicates",
```

Comment:
Is "identify duplicates" clear enough? I think it could be clearer with "group", "index" or "key", but I might be biased. Thoughts, @ostephens and @thadguidry?

Reply:
@ostephens @thadguidry Can you share your inputs on this

Reply:
"Pick columns used to identify duplicate rows" is clear but:
1. it states what picking columns is used for.
2. it clarifies that duplicate rows will be identified and located.
3. but does not say anything about removal built-in, but the modal title does.

I'm proposing:
"Pick columns used to identify and remove duplicate rows" would handle all 3.

I wouldn't worry about longer phrases that bring clarity since text wrapping or smaller font sizes help when horizontally constrained, which we are not in this modal.

Reply:
```suggestion
    "core-dialogs/remove-duplicate-rows-col-select": "Select columns used to identify duplicate rows",
```